src/components/modes/student/StudentView.js

```diff
@@ -28,75 +27,72 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-// const timerType = "backward"
-// const startValue = timerType === "forward" ? 0 : 4800000
+const GridItem = ({ timeValue, timeUnit }) => (
+  <Grid item xs={3} style={{ width: '100%' }}>
+    <Paper variant="outlined" align="center">
+      <Typography variant="h1" color="primary">
+        {timeValue}
+      </Typography>
+      {' '}
+      {timeUnit}
+    </Paper>
+  </Grid>
+);
 
-export const StudentView = ({ t }) => {
+export const StudentView = ({ t, orientation }) => {
   const [value, setValue] = React.useState('forward');
 
-  const handleChange = (event) => {
+  const handleChange = event => {
     setValue(event.target.value);
   };
   const classes = useStyles();
+  const direction = orientation === 'vertical' ? 'column' : 'row';
   return (
     <div>
       <Timer
-        initialTime={value === "forward" ? 0 : 4800000}
+        initialTime={value === 'forward' ? 0 : 4800000}
         startImmediately
-
-
         direction={value}
       >
         {({ start, pause, reset }) => (
           <div className={classes.root}>
-            <Grid container spacing={3} style={{width: "100%"}}>
+            <Grid container spacing={3} style={{ width: '100%' }}>
```

Comment:
The style tag should be avoided. Instead, pass a `className={classes.gridContainer}` and add the styles in the styles object.